interface/portraits/portraits_highelfundead.gfx

```diff
@@ -0,0 +1,361 @@
+# hair, mouth, chin, nose, eyes, cheek, ear, neck
+spriteTypes = {
+
+	### highelf MALE ####
+
+	# CULTURE_INDEX: 0 = clothes behind, 1 = headgear behind, 2 = clothes, 3 = headgear mid, 4 = clothes infront, 5 = headgear
+
+	# highelf Male
+	portraitType = {
+		name = "PORTRAIT_highelfundeadgfx_male"
```

Comment:
Move portrait declarations to `portraits_highelf` since they share everything except 3-2 sprites and eye colors.